src/main/scala/com/fulcrumgenomics/bam/ErrorRateByReadPosition.scala

```diff
@@ -25,266 +25,234 @@
+    |a row per read (first of pair, second of pair and unpaired) per position in read with the total number
+    |of bases observed, the overall error rate, and the rate of each kind of substitution error.
     |
-    |If a reference is given, the sort order must be coordinate.  If no reference is given, the MD tag must be present
-    |for every mapped read and computed correctly, and no sort order is required.
+    |Analysis can be restricted to a set of intervals via the --intervals option. Genomic positions can be
+    |excluded from analysis by supplying a set of variants (either known variants in the sample or a catalog
+    |of known variants such as dbSNP).  For data believed to have low error rates it is recommended to use
+    |both the --intervals and --variants options to restrict analysis to only regions expected to be
+    |homozygous reference in the data.
     |
-    |The output metrics text file will have the extension 'error_rate_by_read_position.txt'.  The output file will
-    |contain a row per logical read and per position in that read.  The 'read_number' will be 0 for fragments, 1 for the
-    |first end of a pair, and 2 for the second end of a pair. Both the error rate and number of bases observed will be
-    |provided.  Soft-clipped bases, and unmapped, secondary, supplementary, and vendor failing reads are ignored.
-    |Optionally, duplicates are ignored.
+    |The following are reads / bases are excluded from the analysis:
+    |  - Unmapped reads
+    |  - Reads marked as failing vendor quality
+    |  - Reads marked as duplicates (unless --include-duplicates is specified)
+    |  - Secondary and supplemental records
+    |  - Soft-clipped bases in records
+    |  - Reads with MAPQ < --min-mapping-quality (default: 20)
+    |  - Bases with base quality < --min-base-quality (default: 0)
+    |  - Bases where either the read base or the reference base is non-ACGT
     |
-    |Mismatches and optionally insertions are considered errors, while deletions and non-ACGT bases are ignored.
-  """,
-  group = ClpGroups.SamOrBam)
+    |If R's Rscript utility is on the path and ggplot2 is installed in the R distribution then a PDF
+    |of error rate plots will also be generated.
+  """
+  )
 class ErrorRateByReadPosition
 ( @arg(flag="i", doc="Input BAM file.") val input: PathToBam,
-  @arg(flag="o", doc="Output metrics prefix. If not given, will use the input BAM basename as a prefix.")
-  val output: Option[PathPrefix] = None,
-  @arg(flag="r", doc="Reference fasta. If not given, will use the MD tag.") val ref: Option[PathToFasta] = None,
+  @arg(flag="o", doc="Output metrics prefix. If not given, will use the input BAM basename.") val output: Option[PathPrefix] = None,
+  @arg(flag="r", doc="Reference sequence fasta file.") val ref: PathToFasta,
+  @arg(flag="v", doc="Optional file of variant sites to ignore.") val variants: Option[PathToVcf] = None,
```

Comment:
We could, but that would be markedly more complicated for, IMHO, pretty much no benefit.